src/app/app.module.ts

```diff
@@ -6,21 +6,16 @@ import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { CreateButtonComponent } from './components/create-button/create-button.component';
 import { AddCardComponent } from './components/add-card/add-card.component';
-import { NewTaskInputDirective } from './directives/new-task-input.directive';
 import { TaskButtonComponent } from './components/task-button/task-button.component';
 
 import { ConstantsService } from 'src/app/services/constants.service';
 import { TaskService } from 'src/app/services/task.service';
 import { UtilityService } from 'src/app/services/utility.service';
+import { NewTaskInputDirective } from './directives/new-task-input.directive';
+
 
 @NgModule({
-  declarations: [
-    AppComponent,
-    CreateButtonComponent,
-    AddCardComponent,
-    NewTaskInputDirective,
-    TaskButtonComponent
-  ],
+  declarations: [AppComponent, CreateButtonComponent, AddCardComponent, NewTaskInputDirective, TaskButtonComponent],
```

Comment:
Please keep the structure you removed. If this is linter's doing, I would try to config the linter.